pkg/yurthub/proxy/local/local.go

```diff
@@ -173,8 +176,9 @@ func (lp *LocalProxy) localWatch(w http.ResponseWriter, req *http.Request) error
 	timeout := time.Duration(0)
 	if opts.TimeoutSeconds != nil {
 		timeout = time.Duration(*opts.TimeoutSeconds) * time.Second
-	} else {
-		return nil
+	}
+	if timeout == 0 && lp.minRequestTimeout > 0 {
+		timeout = time.Duration(float64(lp.minRequestTimeout) * (rand.Float64() + 1.0))
```

Comment:
Do we set seed before calling rand?

Reply:
From yurthub src code, cmd/yurthub/yurthub.go already set seed.
`func main() {
	rand.Seed(time.Now().UnixNano())`

Reply:
Thanks